reproducibility_project/src/engines/cassandra/project.py

```diff
@@ -376,6 +376,71 @@ def run_cassandra(job):
         )
 
 
+@Project.operation
+@Project.pre(cassandra_complete)
+@Project.post(lambda job: "mean_energy_box1" in job.document)
+def statistics(job):
+    """Compute statistical quantities for each job."""
+    import ele
+    from unyt import angstrom, g, mole
+
+    proplist = [
+        "energy_total",
+        "volume",
+        "nmols",
+        "pressure",
+        "density",
+        "mass_density",
+        "energy_angle",
+        "energy_dihedral",
+        "energy_intravdw",
+        "energy_intraq",
+        "energy_inter",
+        "energy_intervdw",
+        "energy_lrc",
+        "energy_interq",
+        "energy_recip",
+        "energy_self",
+        "enthalpy",
+    ]
+
+    NA = 6.032e23
+
+    C = ele.element_from_symbol("C")
+    H = ele.element_from_symbol("H")
+
+    mw = C.mass + H.mass * 4
+    mw *= g / mole
```

Comment:
Why do you need the MW?  Cassandra already knows the molecular weight (it's also included in the parmed structures given to MoSDeF Cassandra) and outputs the mass density, so you can just use the output mass density in the prp file instead of converting from molecular density.  The way this is written, it would only work for methane.  See my suggested change later in the file.

Reply:
Sounds good!